### ISSUE TITLE
VectorMap - Annotations

### ISSUE THREAD

**Issue by Krijovnick** [DevExpress/DevExtreme#14182]
## The Problem

Previously, we introduced annotations in the Chart and Polar Chart widgets. However, annotations are still unavailable for the VectorMap. 

## The Proposed Solution

In DevExtreme v20.2, we are going to bring annotations to the VectorMap, using a similar approach as in other data visualization widgets. There are two mechanisms for annotation placement: 

### Anchor an annotation to map coordinates 
```
<dxi-annotation
    [coordinates]="[-97.514839, 35.466179]"
/>
```
![Map coordinates](https://user-images.githubusercontent.com/8238559/88664155-49a72380-d0e5-11ea-820c-50e25f5600e7.png)

### Use pixel coordinates
```
<dxi-annotation
    [x]="70"
    [y]="120"
/>
```
![Pixel coordinates](https://user-images.githubusercontent.com/8238559/88664157-4ca21400-d0e5-11ea-8c73-cb46b1bd695c.png)

### Live Sandbox

[Angular](https://codesandbox.io/s/immutable-brook-gqdl1)

### Take a Quick Poll

[How do VectorMap annotations meet your needs?](https://forms.gle/JWvXZdrmK3yB8b8V7) 

## Get Notified of Updates

Subscribe to this thread or to our [Facebook](https://www.facebook.com/DevExpress.DevExtreme/) and [Twitter](https://twitter.com/devextreme) accounts for updates on this topic.

**Comment by Stranger52** [comment]
Why not use both types of binding based on the data type? double - coordinates, int - pixels. Or explicitly define what the given coordinate is in.

**Comment by Krijovnick** [comment]
@Stranger52 ,
Since this approach uses data of different types (map coordinates - double, map pixels - integer), it may lead to issues. Map coordinates can be integers too, and their type can be changed to double over time.

**Comment by MaximKudriavtsev** [comment]
Thanks to everybody who gave us early feedback on this feature. The [Vector Map Annotations](https://js.devexpress.com/Demos/WidgetsGallery/Demo/VectorMap/CustomAnnotations/) feature is now available in the [v20.2 release](https://github.com/DevExpress/DevExtreme/releases/tag/20.2.3). I'm closing this thread. In the case of bugs or questions, feel free to create new [GitHub issues](https://github.com/DevExpress/DevExtreme/issues) or tickets in our [Support Center](https://www.devexpress.com/Support/Center/).
